docs/x509/verification.rst

```diff
@@ -76,7 +76,10 @@ the root of trust:
     >>> with open(certifi.where(), "rb") as pems:
     ...    store = Store(load_pem_x509_certificates(pems.read()))
     >>> builder = PolicyBuilder().store(store)
-    >>> builder = builder.time(verification_time)
+    >>> # If significant time passes between the PolicyBuilder is initialized
+    >>> # and the time the validation is perform, you will need to update
+    >>> # the internal clock.
+    >>> builder = builder.time(datetime.now())
```

Comment:
We run these as doc tests and this will fail when the cert expires, so we can'd do that.

I'd suggest just doing a comment explaining this, without changing the code.